src/lib/client.js

```diff
@@ -185,6 +202,18 @@ class Client extends EventEmitter {
 
     return this.plugin.send(transfer)
   }
+
+  /**
+   * Get the list of connector URIs.
+   * @returns {Promise.<URI[]>}
+   */
+  getConnectors () {
```

Comment:
Is this called often or just on startup? If it's called frequently it might be better to just check the ledger's info on startup and store the connectors array, like we do with the `prefix`